src/server/pkg/worker/api_server.go

```diff
@@ -428,3 +430,8 @@ func (a *APIServer) Process(ctx context.Context, req *ProcessRequest) (resp *Pro
 		Tag: &pfs.Tag{tag},
 	}, nil
 }
+
+func (a *APIServer) userCodeEnviron(req *ProcessRequest) []string {
+	return append(os.Environ(), fmt.Sprintf("PACH_JOB_ID=%s", req.JobID))
+	// any way we could get the "COMMIT_ID" of the output ?
```

Comment:
This would be tricky because that `COMMIT_ID` doesn't exist until after the job has finished and the commit can be created.